src/model/Column.ts

```diff
@@ -386,6 +387,13 @@ export default class Column extends AEventDispatcher {
   }
 
   rendererType(): string {
+     if(this.desc.type.constructor === Array) {
+      return this.desc.type[0];
+    }
     return this.desc.type;
   }
+
+  setRendererType(type:string) {
+    this.fire([Column.EVENT_RENDERER_TYPE_CHANGED, Column.EVENT_DIRTY_VALUES, Column.EVENT_DIRTY], this.desc.type, this.desc.type = type);
```

Comment:
as once said the description `desc` should be considered readonly. Descriptions are used as blueprints for creating new columns. Manipulating a description will have unknown side effects, if another instance of the same column is created.